src/navigation/HomeContainer/HomeContainer.tsx

```diff
@@ -72,17 +79,40 @@ function HomeContainer(): React.JSX.Element {
   useLayoutEffect(() => {
     dispatch(hideSplashScreen(true)); // dismiss if not already dismissed when home screen shown.
   });
-
-  return (
-    <Stack.Navigator>
-      <Stack.Screen
-        name="Bookings"
-        component={BookingScreen}
-        options={{
-          headerRight: logoutButton,
-        }}
-      />
-    </Stack.Navigator>
-  );
+  console.log(featureFlags.tabBarEnabled);
+  if (featureFlags.tabBarEnabled === true) {
+    console.log('========');
+    console.log('TAB BAR ENABLED');
```

Comment:
First time reviewing the Wharf spaces app so I'm not sure if this is standard but are these console.logs needed? It seems like either they've accidentally been left in or they would clog up the console. Seems a little superfluous 😄

Reply:
Yeah I need to remove these. I have done in my current WIP that I need to commit. They were in when I was struggling to get the tests to work